google_nest_sdm/google_nest.py

```diff
@@ -178,56 +185,75 @@ def CreateCreds(args) -> Credentials:
     return creds
 
 
-def PrintStructure(structure):
-    print(f"id: {structure.name}")
-    print("traits:")
-    for (trait_name, trait) in structure.traits.items():
-        print(f"  {trait_name}: {trait._data}")
-    print("")
-
+def PrintStructure(structure, output_type):
+    if output_type == "json":
+        print(structure.toJson())
+    elif output_type == "yaml":
```

Comment:
Would you say the yaml output looks similar enough to the console output that it would make a good default?

Reply:
They are quite similar, the only difference being the YAML output has more information.

Reply:
Up for removing the existing one and replacing it with yaml as the default?

One other options is to return the raw dict and do the json/yaml conversion in the printing tool only.  What do you think?

Reply:
Sure.  To remove the `toJson` and `toYaml` and do those conversions in the `Printxxxxx` methods?

Reply:
Yeah, that was what I was thinking, leaving the caller to have to figure out the text specific formatting stuff.  What to you think?

Reply:
@allenporter Updated.